idx/memory/memory.go

```diff
@@ -1300,7 +1302,7 @@ DEFS:
 
 	statMetricsActive.Add(-1 * len(pruned))
 
-	if orgId == -1 {
+	if orgId == idx.OrgIdPublic {
```

Comment:
Doesnt look like the use of `-1` had anything to do with the PublicOrg.  So i think instead this should be 
```
if orgId == 0 {
```